### PR TITLE
xGov-37 - NFT Lending – dApp

### PR DESCRIPTION
**Pull request by palace22**

### CODE REVIEW

File: Proposals/xgov-37.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+title: NFT Lending – dApp
+author: Benedetto Biondi <benedetto@folks.finance>, Gidon Katten (@gidonkatten), Stefano Faieta (@stefanofa), Ibu Karel <ibu@folks.finance>, Giuseppe Palazzolo (@palace22)
+company_name: Folks Finance
+category: dApps
+focus_area: Defi,NFT
```
Comment (SudoWeezy) [inline Proposals/xgov-37.md:6]: Only use Defi OR NFT, it will not be usable by the voting tool otherwise

File: Proposals/xgov-37.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+The development of this NFT lending platform will evolve the ecosystem by unlocking the value of tokenized assets and enabling access to liquidity through borrowing fungible tokens. The tools will be open-sourced, meaning any project can integrate this technology and shape it to their needs. Benefits and revenue will proliferate around Algorand’s network. NFT Lending fosters financial inclusion, empowers NFT holders to leverage their assets, and opens up new opportunities for participants within the tokenization space. The long-term applications of this technology are limited only by how much value, and how many different things, can be tokenized.
+
+## Team
+
+[Benedetto Biondi](https://www.linkedin.com/in/benedetto-biondi-030248199/) is the co-founder and CEO of Folks Finance, one of the leading projects on Algorand, offering a full set of DeFi tools for managing Digital assets. Benedetto has accrued 3 years of experience as a Blockchain Project Manager, is a mentor in the Algorand Miami and Europe Accelerators, a DeFi advisor at Climatecoin, and a DeFi lecturer at the course of Economics and Law for Digital Assets at the University of Florence.
```
Comment (SudoWeezy) [inline Proposals/xgov-37.md:38]: External link needs to be formatted using html <a>
eg `<a href="https://www.linkedin.com/in/benedetto-biondi-030248199/">Benedetto Biondi</a>`

File: Proposals/xgov-37.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+[Gidon Katten](https://www.linkedin.com/in/gidonkatten/) joined Folks Finance as CTO in late 2021 after leaving his job at Amazon. Gidon graduated from Imperial College London where he wrote an award winning paper titled “Issuing Green Bonds on the Algorand Blockchain”.
```
Comment (SudoWeezy) [inline Proposals/xgov-37.md:40]: External link needs to be formatted using html <a>
eg `<a href="https://www.linkedin.com/in/gidonkatten/">Gidon Katten</a>`

File: Proposals/xgov-37.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+[Stefano Faieta](https://www.linkedin.com/in/stefanofaieta/) has always been passionate about technology and IT. While pursuing his university degree, he was accepted into exclusive programs such as one focused on startups and innovation, and another on cybersecurity. He graduated with honors in computer science and completed a thesis on the interoperability of smart contracts on the Ethereum network. Initially, he began his career as a frontend developer in traditional finance at banking institutions. However, his persistent desire for experimentation and innovation quickly drove him to merge his two greatest interests - blockchain and frontend technologies.
```
Comment (SudoWeezy) [inline Proposals/xgov-37.md:42]: External link needs to be formatted using html
eg `<a href="https://www.linkedin.com/in/stefanofaieta/">Stefano Faieta</a>`

File: Proposals/xgov-37.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+[Ibu Karel](https://www.linkedin.com/in/ibukarel/) is the Head of Partnerships at Folks Finance. He focuses on building connections and creating synergies within and beyond the blockchain industry. Ibu also operates as a content creator for Folks Finance, writing articles, formulating marketing strategies, and constructing videos like those found on the Folks Academy.
```
Comment (SudoWeezy) [inline Proposals/xgov-37.md:44]: External link needs to be formatted using html
eg `<a href="https://www.linkedin.com/in/ibukarel/">Ibu Karel</a>`

File: Proposals/xgov-37.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+[Giuseppe Palazzolo](https://www.linkedin.com/in/giuseppe-palazzolo-08/) is a blockchain and back-end developer with 3 years of professional experience. He contributed to the development of Folks Feed Oracle and undertook multiple projects on Algorand. He holds a master's degree in Computer Engineering with a focus on Security and Data Analytics from the University of Florence.
```
Comment (SudoWeezy) [inline Proposals/xgov-37.md:46]: External link needs to be formatted using html
eg `<a href="https://www.linkedin.com/in/giuseppe-palazzolo-08/">Giuseppe Palazzolo</a>`

**Review by SudoWeezy — CHANGES_REQUESTED** [review]
As you did not allow us to edit your proposal, you will need to make the requested changes yourself :)

File: Proposals/xgov-37.md (hunk 6 above)
Comment (palace22) [reply Proposals/xgov-37.md]: Thanks @SudoWeezy, updated!

**Review by SudoWeezy — APPROVED** [review]

**Pull request merged**